vars/fcosKola.groovy

```diff
@@ -24,6 +24,9 @@ def call(params = [:]) {
         arch = "--arch=${arch}"
     }
 
+    // Create a unique output directory for this run of fcosKola
+    outputDir = shwrapCapture("mktemp -d ${cosaDir}/fcosKola-XXXXX")
```

Comment:
do we need to worry about cleaning this up at the end?

Reply:
Maybe

```suggestion
    outputDir = shwrapCapture("mktemp -d ${cosaDir}/tmp/fcosKola-XXXXX")
```

instead?

> do we need to worry about cleaning this up at the end?

I think if we put it in the cosa dir's `tmp/` it should be fine to leave around.

Reply:
Done